webscraping/css_stat_generation.py

```diff
@@ -0,0 +1,69 @@
+from webscraping import web_scraper
+import tinycss2
+import json
+
+def get_css_stats(url):
+    stats = {
+        "tags": {},
+        "ids": {},
+        "classes": {}
+    }
+
+    def parse_rule_set(rule):
+        if rule.type == "qualified-rule":
+            prelude = [token for token in rule.prelude if token.type != "whitespace"]
+            if len(prelude) == 1:
+                subject = prelude[0]
+                if subject.type == "ident":
+                    stats["tags"][subject.lower_value] = parse_declarations(rule)
```

Comment:
One thing that we should take into account is tag styles being defined in multiple places. This seems like it would override any prior css for the tag.